sqlglot/helper.py

```diff
@@ -30,3 +31,8 @@ def list_get(arr, index):
 
 def csv(*args, sep=', '):
     return sep.join(arg for arg in args if arg)
+
+
+def camel_to_snake_case(name):
+    name = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
```

Comment:
if we were to have this, we should probably precompile the regex

Reply:
if we do happen to do this change, which i'm not convinced of yet, i think we should require the user to provide the key of the expression so that it's static

Reply:
Great point 👍 I didn't really consider performance implications.

Reply:
Fixed this. Please have a look

Reply:
can we precompile the regex?

Reply:
Done that. Also decided to go with a simpler patter for now.